### DISCUSSION TITLE
New OS

### DISCUSSION THREAD

**Discussion by poquirion** [ComputeCanada/magic_castle#244] [General]
With all the fuzz with RHEL, it is time to test a MC with a Debian or Suse distro? How much work would that involve? What about the selinux implications?

**Comment by cmd-ntrf** [comment]
We currently have a strong dependency on the CentOS/RHEL ecosystem:
- FreeIPA is tightly coupled with RHEL ecosystem since it is developed by RedHat;
- We use Fedora COPR to build and distributes our RPMs.
- All puppet code that install packages supposes yumrepo and rpms.

It is not impossible to move to Debian, but it is not something that can be done quickly. Since the original intent of MC was to replicate a Compute Canada / The Alliance HPC system from a software standpoint, I would suggest we wait for an upstream decision of switching OS before we do any work related to that.
